app/src/main/java/de/tum/in/tumcampusapp/managers/TumManager.java

```diff
@@ -110,14 +110,16 @@ public String addLock(String url, String data) {
         //Try to parse the error
         String msg = "";
         try {
-
+            if(data.contains("<rowset>\n</rowset>")){
```

Comment:
You could use the constant from TUMOnlineRequest here or move it to the Constants file